src/bus/services.rs

```diff
@@ -148,7 +148,7 @@ where
     ) -> Result<(), esb::Error<ServiceId>> {
         endpoints.send_to(
             ServiceBus::Storm,
-            self.identity(),
+            ServiceId::rgbd(),
```

Comment:
We must not send messages from the identity of a different daemon

Reply:
I changed the identity to handle the "transfer consignment file"  in the `storm daemon`. But, according to your comment [here](https://github.com/Storm-WG/storm-node/pull/2#discussion_r973695082), I think it is not the correct approach, and I will try another method.